### PR TITLE
test(dvla): use http and sdk fixtures

### PR DESCRIPTION
**Pull request by aron-bs**
# Pull Request

## Description

- Update DVLA test suites to use http/sdk fixtures
- Add test cases for missing branches
- Add domain-specific test fixtures
- Add path alias to improve imports

**Related Issue:** https://gdsgovukagents.atlassian.net/browse/FLEX-240

## Type of Change

Please check options that are relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Refactor (code change that does not add functionality or fix a bug)
- [x] Code cleanup (formatting, renaming)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [ ] Other (please explain)

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so others can reproduce.

- [x] Unit tests
- [x] Integration tests
- [ ] Manual testing (include instructions below)

## Checklist

- [x] I have run the pre-commit
- [x] I have run all necessary tests
- [x] I have updated/added all necessary tests
- [x] I have added or updated documentation
- [x] I have performed a self-review of my code
- [x] I have commented my code in hard-to-understand areas
- [x] I have refactored my code to be more readable, maintainable and removed duplications.
- [x] I have added guards around invalid inputs and edge cases such as nulls and undefined

### CODE REVIEW

File: domains/dvla/src/handlers/v1/customer-summary/get.test.ts [hunk 1]
```diff
@@ -1,121 +1,113 @@
...
-  const testLinkingId = "550e8400-e29b-41d4-a716-446655440000";
-  const testCustomerId = "6ba7b810-9dad-11d1-80b4-00c04fd430c8";
+  const endpoint = "/customer-summary";
 
-  beforeEach(() => {
-    vi.stubEnv("flexDvlaTestUser", testLinkingId);
+  it.beforeEach(() => {
+    // TODO: Is this needed?
+    // env.set({ flexDvlaTestUser: dvlaTestUser });
   });
```
Comment (pazsaragi) [inline domains/dvla/src/handlers/v1/customer-summary/get.test.ts]: Remove if not?

Comment (aron-bs) [reply domains/dvla/src/handlers/v1/customer-summary/get.test.ts]: Left the comments in to get thoughts incase there was a reason it was added in the first place but I'll remove them

**Review by nathanberry97 — APPROVED** [review]
